### DISCUSSION TITLE
Copy buffer not subject to undo commands

### DISCUSSION THREAD

**Discussion by jovlinger** [zed-industries/zed#12305] [Feature Requests]
If I have a piece of text in the copy buffer, and then flub it, and accidentally copy instead of paste, I cannot fix this via clover-Z undo. That does not restore the copy buffer.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
